core/Forms/Forms.php

```diff
@@ -31,4 +34,66 @@ private function checkInput($data) {
         return htmlentities($data);
     }
 
+    public function notEmpty($field) {
```

Comment:
Are you sure that this is semantically correct function not Empty returns if string if empty?